README.md

```diff
@@ -35,6 +45,43 @@ ddf = dask_bigquery.read_gbq(
 
 ddf.head()
 ```
+## Example: write to BigQuery
+
+With default credentials:
+
+```python
+import dask
+import dask_bigquery
+
+ddf = dask.datasets.timeseries(freq="1min")
+
+res = dask_bigquery.to_gbq(
+    ddf,
+    project_id="my_project_id",
+    dataset_id="my_dataset_id",
+    table_id="my_table_name",
+)
+```
+
+With explicit credentials:
+
+```python
+from google.oauth2.service_account import Credentials
+
+# credentials
+creds_dict = {"type": ..., "project_id": ..., "private_key_id": ...}
+credentials = Credentials.from_service_account_info(info=creds_dict)
+
+res = to_gbq(
+    ddf,
+    project_id="my_project_id",
+    dataset_id="my_dataset_id",
+    table_id="my_table_name",
+    credentials=credentials,
+)
+```
+
+`to_gbq` writes intermediary Parquet to Google Storage bucket. Default bucket name is set to "dask-bigquery-tmp". You can provide a diferent bucket name by setting the parameter: `bucket="my-gs-bucket"`.
```

Comment:
Should be "to a Google Storage bucket", right?

Might be nice to say something explicit about cleanup, too.